types/yup/yup-tests.ts

```diff
@@ -680,7 +680,7 @@ const definitionBC: yup.ObjectSchemaDefinition<BC> = {
     b: yup.string(),
     c: yup.number(),
 };
-const combinedSchema = yup.object(definitionAB).shape(definitionBC); // $ExpectType ObjectSchema<Shape<AB, BC>>
+const combinedSchema = yup.object(definitionAB).shape(definitionBC); // $ExpectType ObjectSchema<{ a: string; b: string; } & BC>
```

Comment:
The linter didn't like `ObjectSchema<Shape<AB, BC>>` here, even though the types are equivalent.  No idea why.